test/unit/modules/util/next-tick.spec.js

```diff
@@ -0,0 +1,25 @@
+import { nextTick } from 'core/util/env'
+
+describe('nextTick', () => {
+  it('accepts a callback', done => {
+    nextTick(done)
```

Comment:
Because I didn't want to encourage people to mix the two styles (passing a callback vs chaining a promise).